privacyidea/lib/policy.py

```diff
@@ -314,6 +314,7 @@ class ACTION(object):
     TOKENREALMS = "tokenrealms"
     TOKENTYPE = "tokentype"
     TOKENINFO = "tokeninfo"
+    TOKENINFO_UI_EXCLUDE = "tokeninfo_ui_exclude"
```

Comment:
The naming is not good, you are removing the tokeninfo on the API level, not only the UI.

Reply:
i can rename it to `TOKENINFO_API_EXCLUDE` do you think this is a good name?